release/nightly_tests/dataset/benchmark.py

```diff
@@ -112,10 +115,8 @@ def run_iterate_ds(
             if isinstance(batch, dict):
                 feature_lengths = {k: len(batch[k]) for k in batch}
                 batch_size = max(feature_lengths.values())
-                continue
```

Comment:
Should this have been removed before?

Reply:
yeah, this was a bug i missed. but reporting for release tests seems unaffected, the specific metrics were not used before